### DISCUSSION TITLE
What are the best pretrained models? Where to find them? What are the best model to get general purpose objects - like- cars, homes, people, doors and so on.

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [ultralytics/ultralytics#6983] [General]
What are the best pretrained models? Where to find them? What are the best model to get general purpose objects - like- cars, homes, people, doors and so on.

Is there a special site or repository to find pretrained models?

**Comment by pderrenger** [comment]
@martinenkoEduard the best pretrained models for general-purpose object detection, including cars, homes, people, doors, etc., are typically the ones trained on large and diverse datasets like COCO. For YOLOv8, you can find the pretrained models directly within the repository. These models have been trained on the COCO dataset, which contains 80 different classes that cover a wide range of common objects.

To get started with a pretrained YOLOv8 model, you can simply load it using the following Python code:

```python
from ultralytics import YOLO

# Load a pretrained YOLOv8 model
model = YOLO('yolov8n.pt')  # Replace 'yolov8n.pt' with the specific model you want to use
```

The model names typically end with a suffix that indicates their size and complexity, such as 'n' for nano, 's' for small, 'm' for medium, and 'l' for large. Larger models usually offer higher accuracy but require more computational resources.

For the most up-to-date information on available models and their performance, please refer to the Ultralytics Docs. There you'll find details on how to use these models and further instructions on training and evaluation. 📘🚀
